src/controllers/botapi/index.ts

```diff
@@ -77,8 +78,8 @@ export async function processAction(req, res) {
   }
 
   try {
-    const r = await finalAction(a)
-    success(res, r)
+    await finalAction(a)
+    success(res, {})
```

Comment:
I'm lost here does finalAction always return `{}`?

Reply:
finalAction returns void, nothing. i remove the const because assigning void to a variable is useless, at that point success only accepted json, so gave it an empty object (now it also accepts a string because it was used in other places)